### PR TITLE
[IMP] impakta: Welcome sale_order_report_product_image

### PR DESCRIPTION
**Pull request by JoseIbarraHdz**
The module sale order report product image is now working on v17.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -2,3 +2,7 @@
 	path = impakta
 	url = git@git.vauxoo.com:vauxoo/impakta.git
 	branch = 17.0
+[submodule "sale-reporting"]
+	path = sale-reporting
+	url = git@github.com:Vauxoo/sale-reporting.git
+	branch = 17.0-task-81483-JoseIbarraHdz
```
Comment (edy1192) [inline .gitmodules]: This must be the stable branch (17.0) please.

**Review by edy1192 — CHANGES_REQUESTED** [review]

**Review by edy1192 — APPROVED** [review]

**Comment by JoseIbarraHdz** [comment]
Hi @hugho-ad Edy approved the changes and the client give us the VoBo to merge, can you merge the PR please?

**Comment by hugho-ad** [comment]
@JoseIbarraHdz  

validates with the requested these changes, when can we do the release changes to production please

**Comment by hugho-ad** [comment]
@JoseIbarraHdz sorry, not  @JesusValdez96

**Pull request merged**